File: src/test/java/MonotonicTest.java
import org.junit.jupiter.api.Test;

import static org.assertj.core.api.AssertionsForClassTypes.assertThat;

public class MonotonicTest {
    private Monotonic monotonic = new Monotonic();

    @Test
    public void given_1_3_2_return_false() {
        int[] nums = new int[]{1,3,2};
        assertThat(monotonic.isMonotonic(nums)).isFalse();
    }
}

File: src/test/kotlin/com/corrigan/MonotonicTest.kt
package com.corrigan

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class MonotonicTest {

    @Test
    fun `given an integer array of size 2 and values 1,2 return true`() {
            assertThat(Monotonic.isMonotonic(intArrayOf(1,2))).isTrue()
    }

    @Test
    fun `given an integer array of size 4 and values 1,2,2,3 return true`() {
        assertThat(Monotonic.isMonotonic(intArrayOf(1,2,2,3))).isTrue()
    }

    @Test
    fun `given an integer array of size 4 and values 6,5,4,4 return true`() {
        assertThat(Monotonic.isMonotonic(intArrayOf(6,5,4,4))).isTrue()

    }

    @Test
    fun `given an integer array of size 3 and values 1,3,2 return true`() {
        assertThat(Monotonic.isMonotonic(intArrayOf(1,3,2))).isFalse()

    }


}